README.md

```diff
@@ -38,12 +38,16 @@ A sample query : `/api/v1/search/bing?query=fossasia&format=xml&num=10`
 
 ## Dependencies
 
-* Python 2.x or Python 3.x
-* [Node.js](https://nodejs.org/en/)
-* [Pip](https://pip.pypa.io/en/stable/installing/)
-* [Flask](http://flask.pocoo.org/)
-* [BeautifulSoup4](https://www.crummy.com/software/BeautifulSoup/bs4/doc/)
-
+* [MongoDB](https://www.mongodb.com)
+* [Python 2.7 and 3.4+](https://python.org) `  ` Best practice: [__pip install -r requirements.txt__](requirements.txt)
```

Comment:
In Installation, `pip install -r requirements.txt` it's already there so it's not needed here

Reply:
@cclauss Also, please update your PR with the latest codebase 😄 and fix conflicts if it appear. Thanks